website/docs/docs/dbt-versions/release-notes/70-Apr-2024/mar-sl-updates.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "dbt Semantic Layer updates and fixes for March 2024"
+description: "March 2024 updates for dbt Semantic Layer & MetricFlow: New export features, enhanced date filters, Tableau, Google Sheets & GraphQL API enhancements, and bug fixes."
+sidebar_label: "Updates and fixes: dbt Semantic Layer"
+sidebar_position: 10
+tags: [Mar-2024]
+date: 2024-04-01
+---
+
+The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and [MetricFlow](/docs/build/about-metricflow).
+
+The following list explains the new features, updates, and fixes for March 2024 in more detail.
+
+## dbt Semantic Layer and MetricFlow
```

Comment:
I realize this is following the format of a previous release note, but would it be more in line with our style guide to let the title of the page handle this and then have the individual sections below (New features, Updates, Bug fixes) be H2 or H3s?